pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -3708,6 +3708,28 @@ Bottom Left.`,
 			tree.VolatilityImmutable,
 		),
 	),
+	"st_flipcoordinates": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geometry", types.Geometry}},
```

Comment:
nit: new line per argument for multiline args, i.e.

```
Types: tree.ArgTypes{
				{"geometry", types.Geometry},
},
```

Reply:
Done.